core/light-client/src/lib.rs

```diff
@@ -323,14 +323,14 @@ where
 		header: Block::Header,
 		justifications: Option<Justifications>,
 	) -> Result<(), Error> {
-		let mut relay = self.tracked_relays.get_mut(&relay_id).ok_or(Error::NoSuchRelayExists)?;
+		let relay = self.tracked_relays.get_mut(&relay_id).ok_or(Error::NoSuchRelayExists)?;
 
 		if relay.last_finalized_block_header.hash() != *header.parent_hash() {
 			return Err(Error::HeaderAncestryMismatch)
 		}
 		let ancestry_proof = vec![];
 
-		Self::apply_validator_set_change(&mut relay, &header);
+		Self::apply_validator_set_change(relay, &header);
```

Comment:
Are these changes motivated by new clippy warnings?

Reply:
Yes, all code changes except Cargo.toml and Cargo.lock are due to clippy.